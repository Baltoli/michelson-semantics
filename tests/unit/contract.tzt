code { CONTRACT unit } ; 
input { Stack_elt address "12345" } ;
output { Stack_elt (option (contract unit)) (Some "12345") } ;
other_contracts { Elt "12345" unit } 
