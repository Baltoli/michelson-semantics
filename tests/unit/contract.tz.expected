<generatedTop>
  <k>
    .
  </k>
  <paramtype>
    #NotSet
  </paramtype>
  <storagetype>
    #NotSet
  </storagetype>
  <stack>
    Stack ( ListItem ( Some #Contract ( #Address ( 12345 ) , unit ) ) )
  </stack>
  <balance>
    #Mutez ( 0 )
  </balance>
  <amount>
    #Mutez ( 0 )
  </amount>
  <now>
    #Timestamp ( -1 )
  </now>
  <myaddr>
    #Address ( -1 )
  </myaddr>
  <knownaddrs>
    #Address ( 12345 ) |-> #Contract ( #Address ( 12345 ) , unit )
  </knownaddrs>
  <sourceaddr>
    #Address ( -2 )
  </sourceaddr>
  <senderaddr>
    #Address ( -3 )
  </senderaddr>
  <chainid>
    #ChainId ( 0 )
  </chainid>
</generatedTop>
